src/views/AvailableTeams.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="cont">
+      <h1>Available Teams</h1>
+      <p class="basicText">You do not currently belong to any team</p>
+      <router-link
+      :to="{ name: 'TeamView' }"
+      v-for="team in teams"
+      :key="team">
+        <p class="team">{{ team.name }}</p>
+      </router-link>
+      <b-button class="create" @click="createTeam">Create New Team</b-button>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import VueRouter from 'vue-router';
+
+Vue.use(VueRouter);
+
+export default {
+  name: 'availableTeams',
+  data() {
+    return {
+      teams: [
```

Comment:
Will this data eventually just live in Vuex? If so, can you put it there now? If it's not decided, this is fine.

Reply:
No, it will be called from the back end on